core/domain/exp_jobs_one_off.py

```diff
@@ -491,7 +493,7 @@ def map(item):
 
         html_list = exploration.get_all_html_content_strings()
 
-        err_dict = html_cleaner.validate_textangular_format(html_list)
+        err_dict = html_cleaner.validate_rte_format(html_list, 'text-angular')
```

Comment:
Make text-angular into a named constant: RTE_FORMAT_TEXTANGULAR. Use that throughout.

Reply:
Done.